projects/cds/src/profiletag/services/profile-tag.injector.service.ts

```diff
@@ -63,4 +66,14 @@ export class ProfileTagInjectorService {
       })
     );
   }
+
+  private notifyEcOfLoginSuccessful(): Observable<boolean> {
+    return this.spartacusEventTracker.loginSuccessful().pipe(
+      switchMap((_) => {
```

Comment:
We could simplify this a bit by using `switchMapTo`: `switchMapTo(this.cdsBackendConnector.notifySuccessfulLogin().pipe(mapTo(true)))`. 
Functionality wise, it's exactly the same as before, so the decision is completely up to you :)